src/olympia/pages/urls.py

```diff
@@ -48,7 +48,7 @@
         name='pages.sunbird'),
     url('^sunbird/', lambda r: perma_redirect(reverse('pages.sunbird'))),
 
-    url('^shield-study-1/',
```

Comment:
Looks like you're renaming the template in this PR. I suppose the only thing is if anyone was linking the previous URL. If 404s aren't a problem to you then removing this is OK assuming there's not a bunch of backlinks pointing at it.